polkadot/node/core/backing/src/lib.rs

```diff
@@ -1946,6 +1986,13 @@ async fn handle_second_message<Context>(
 		Some(r) => r,
 	};
 
+	// Just return if the local validator is disabled. If we are here the local node should be a
+	// validator but defensively use `unwrap_or(false)`) to continue processing in this case.
+	if rp_state.table_context.local_validator_is_disabled().unwrap_or(false) {
```

Comment:
same question here as above
```suggestion
	// validator but defensively use `unwrap_or(false)` to continue processing in this case.
	if rp_state.table_context.local_validator_is_disabled().unwrap_or(false) {
```